flowapi/Dockerfile-dev

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM python:3.7
+
+WORKDIR /flowapi
+COPY Pipfile* /
+RUN pip install pipenv && pipenv install --dev
+COPY app /flowapi/app
+RUN mkdir /var/log/flowapi
+VOLUME /var/log/flowapi
+CMD pipenv run watchmedo auto-restart --recursive --patterns="*.py" --directory="." pipenv run ./app/start.sh
```

Comment:
Not objecting, but what's the rationale for using this as opposed to the autoreload that's built into flask & quart?

Reply:
Good point. For the time being it was just expediency to get it working quickly (and I was going to address this in a subsequent PR when adding proper dev/prod configurations for flowapi) but there is no reason to not already use this here. I'll switch it over.

Reply:
On that note, is the `CONFIG` variable (which is set in the `Dockerfile` and also in `flowapi/tests/unit/conftest.py`) actually used anywhere, or can I remove this and only set `FLASK_ENV=development`?

Reply:
Also, the `app/start.sh` script runs the app via `hypercorn`. Does this actuallly pick up the `FLASK_ENV` variable and trigger a restart of the app upon source code changes?

Reply:
Hmm, a quick experiment seems to suggest that setting `QUART_ENV=development` in the `Dockerfile-dev` (before calling `pipenv run app/start.sh`) is not sufficient to enable a live reload. Not sure what I'm missing, but perhaps worth investigating later and addressing in a separate PR?